client/src/components/Login.js

```diff
@@ -88,12 +98,21 @@ function Login() {
 								color="primary"
 								onClick={() => dispatch(login({ email, password }))}
 							>
-								Primary
+								Login
+							</Button>
+							<br />
+							<Button
+								component={Link}
+								to={"/register"}
+								className={classes.input}
+								variant="contained"
+								color="primary"
+							>
+								Register
 							</Button>
```

Comment:
Add a register button to on the login page so they can go and register a new account